src/utils/fetchDefaultEpicContent.test.ts

```diff
@@ -0,0 +1,39 @@
+import { fetchDefaultEpicContent } from './fetchDefaultEpicContent';
```

Comment:
cc @andre1050 and @tjmw - what are people's preferences on:

a) naming these files? (with Andre earlier I'd assumed we'd go for nouns as good names here - the reason being that it makes it easier to add additional functions later, i.e. they are more like mini-modules or objects, which typically have noun naming

b) utils folder? - I don't have strong views on this, generally I prefer a name other than `utils` though (or things like `common` - as `common` in Frontend still makes me sad).

Reply:
Hey @nicl both good questions!

For a) yep I can see the point here. I agree that's the direction I'd expect the naming to go eventually. The name as it is now definitely doesn't invite adding additional functions. I guess the danger is going with something too generic before figuring out what we want this module to be which can end up being misleading. So we could rename now or put that decision off until we have a concrete use case. I don't feel too strongly either way.

b) yeah I think we should look to split `utils` up into several more specifically named folders. I've definitely seen `utils` become a bit of a dumping ground in projects in the past. Perhaps the file added here belongs in `src/api` or similar.

Reply:
I ended up reorganizing and renaming things here a little bit. We've now got `src/api/contributionsApi` where we can perhaps group all API requests we need to make to fetch contributions data. Maybe not perfect, but we can always change in future.